src/functions/updateCartOnChangedProduct/function.ts

```diff
@@ -0,0 +1,49 @@
+import dbContext from '@dbModel/dbContext';
+import CartRow from '@dbModel/tables/cart';
+import * as AWS from 'aws-sdk';
+
+const updateCartOnChangedProduct = async (productId: string): Promise<void> => {
+    const iterator = await dbContext.scan(CartRow, { filter: { type: 'Equals', subject: 'productId', object: productId } });
```

Comment:
Pay attention on 1 Mb limit of scan. If we'll have a scan result that exceed 1 mb of size, this scan will not return all cart rows. Use 
```suggestion
    const paginator = (await dbContext.scan(CartRow, { filter: { type: 'Equals', subject: 'productId', object: productId } })).pages;
```